src/test/kotlin/com/ampnet/projectservice/controller/ProjectControllerTest.kt

```diff
@@ -862,6 +864,44 @@ class ProjectControllerTest : ControllerTestBase() {
         }
     }
 
+    @Test
+    @WithMockCrowdfundUser
+    fun mustBeAbleToGetPersonalProjects() {
+        suppose("There is an organization") {
+            testContext.organization = createOrganization("some org", UUID.randomUUID())
+        }
+        suppose("User is member of organization") {
+            databaseCleanerService.deleteAllOrganizationMemberships()
+            addUserToOrganization(userUuid, testContext.organization.uuid, OrganizationRole.ORG_MEMBER)
+        }
+        suppose("Organization has projects") {
+            testContext.project = createProject("project1", testContext.organization, userUuid)
+            testContext.anotherProject = createProject("project2", testContext.organization, UUID.randomUUID())
+        }
+        suppose("Some other organization has projects") {
+            createProject("project3", organization, UUID.randomUUID())
+            createProject("project4", organization, UUID.randomUUID())
+        }
+        suppose("User is a member of another organization which has no projects") {
+            val organization = createOrganization("another org", UUID.randomUUID())
+            addUserToOrganization(userUuid, organization.uuid, OrganizationRole.ORG_MEMBER)
+        }
+
+        verify("Controller will return user's personal projects") {
+            val result = mockMvc.perform(
+                MockMvcRequestBuilders.get("/project/personal")
+            )
+                .andExpect(status().isOk)
+                .andReturn()
+            val response: ProjectListResponse = objectMapper.readValue(result.response.contentAsString)
+            assertThat(response.projects).hasSize(2)
+            val projects = response.projects
```

Comment:
Can we have projects from different organizations to make the test more accurate?